app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -1813,6 +1813,11 @@ public void hideVRVideo() {
         queueRunnable(this::hideVRVideoNative);
     }
 
+    @Override
+    public void switchPassthrough() {
```

Comment:
I would also change `switchPassthrough` for `togglePassthrough`, or use `switch` but with a bool argument. For me switch passthrough sounds like it always activate it.

Reply:
I precisely thought the opposite :) but not strong opinion. I'm happy with whatever you decide.

Reply:
I do prefer toggle as well

Reply:
ok